curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md

```diff
@@ -47,17 +40,17 @@ tests:
 
 ```html
 <script type="text/javascript">
-  function contour() {
+  function runSample() {
     let mat = cv.imread("imageSrc");
     let edged = new cv.Mat();
     let contours = new cv.MatVector();
     let hierarchy = new cv.Mat();
 
     cv.Canny(mat, edged, 100, 200);
+    cv.imshow("canvasCanny", edged);
 
-    cv.findContours(edged, contours, hierarchy, cv.RETR_CCOMP, cv.CHAIN_APPROX_SIMPLE);
-
-    cv.drawContours(mat, contours, -1, new cv.Scalar(0, 255, 0, 255));
+   
+    cv.drawContours(mat, contours, -1, new cv.Scalar(255, 0, 0, 255),2);
```

Comment:
Let's ask user to implement this function if you don't mind.

Reply:
Sorry,  I copy this line, but don't delete